src/lib/components/PreFooter.svelte

```diff
@@ -93,14 +94,23 @@
                         <div class="info text-caption font-medium">per org/month</div>
                     </div>
                     <p class="web-strip-plans-info text-caption font-medium">
-                        For pro developers and production projects that need the ability to scale.
+                        For teams that handle more complex and large projects and need more control
+                        and support.
                     </p>
-                    <button
-                        class="web-button is-full-width-mobile is-secondary web-u-cross-child-end"
-                        disabled
+                    <a
+                        href={`${PUBLIC_APPWRITE_DASHBOARD}/console?type=create&plan=tier-2`}
+                        class="web-button is-full-width-mobile web-u-cross-child-end"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        on:click={() =>
+                            trackEvent({
+                                plausible: {
+                                    name: 'Get started Scale plan'
```

Comment:
do we have that event somewhere else already?

Reply:
No clue :sweat_smile: